Update ansible_tips_tricks.rst (#275)
ansible/ansible-documentation: docs/docsite/rst/tips_tricks/ansible_tips_tricks.rst

-----------------------

Testing changes in a staging environment before rolling them out in production is always a great idea.

> Your environments need not be the same size and you can use group variables to control the differences between those environments. you can also check for any syntax errors in staging environment using Ansible lint 

Reviewer: ```suggestion
Your environments need not be the same size and you can use group variables to control the differences between those environments.
You can also check playbooks for syntax errors in staging environments with Ansible Lint, as follows:
```

Reviewer 2: There's no need to use ansible-lint for checking for syntax errors. You can simply use `ansible-playbook --syntax-check` (which is what ansible-lint uses under the hood).

I would rather reference that instead of taking out the big hammer (i.e. ansible-lint).

> Your environments need not be the same size and you can use group variables to control the differences between those environments. you can also check for any syntax errors in staging environment using flag --syntax-check 

Reviewer 3: ```suggestion
Your environments need not be the same size and you can use group variables to control the differences between those environments. you can also check for any syntax errors in staging environment using flag ``--syntax-check``
```
